### PR TITLE
Upgrade bioformats to 5.5.3

### PR DESCRIPTION
**Pull request by mcquin**
* Upgrade loci_tools.jar to 5.5.3
* Use latest schema
* Remove tests from package and
  * Use conftest to start/stop javabridge
  * Declare test dependencies
  * Install test dependencies on Travis

### CODE REVIEW

**Comment by mcquin** [comment]
Resolves #82

File: bioformats/__init__.py [hunk 1]
```diff
@@ -89,16 +89,6 @@
 
 from . import omexml
 
-def init_logger():
```
Comment (mcquin) [inline bioformats/__init__.py]: TODO: Configure a logger.

File: bioformats/tests/resources/tiff.xml [hunk 2]
```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OME xmlns="http://www.openmicroscopy.org/Schemas/OME/2013-06"
```
Comment (mcquin) [inline bioformats/tests/resources/tiff.xml]: TODO: Use latest schema.

File: bioformats/__init__.py (hunk 1 above)
Comment (0x00b1) [inline bioformats/__init__.py]: 👍

**Comment by nerduno** [comment]
Thank you.  I will try it.

**Pull request merged**